src/api/data/endringsmelding.ts

```diff
@@ -8,7 +8,12 @@ export enum EndringsmeldingType {
     FORLENG_DELTAKELSE = 'FORLENG_DELTAKELSE',
     AVSLUTT_DELTAKELSE = 'AVSLUTT_DELTAKELSE',
     DELTAKER_IKKE_AKTUELL = 'DELTAKER_IKKE_AKTUELL',
-	ENDRE_DELTAKELSE_PROSENT = 'ENDRE_DELTAKELSE_PROSENT'
+	ENDRE_DELTAKELSE_PROSENT = 'ENDRE_DELTAKELSE_PROSENT',
+	TILBY_PLASS = 'TILBY_PLASS',
+	SETT_PAA_VENTELISTE = 'SETT_PAA_VENTELISTE',
+	ENDRE_SLUTTDATO = 'ENDRE_SLUTTDATO'
```

Comment:
Er det sånn at vi ikke kan bruke FORLENG_DELTAKELSE fordi at for kurs så kan sluttdato endres, men det er ikke nødvendigvis snakk om at man deltar lenger enn ellers? F.eks., at datoen kan endres fra 30/4 til 28/4? Bare så jeg skjønner hvorfor vi ikke kan gjenbruke meldingen som finnes :)

Reply:
Det er en mulighet for at rent teknisk så kan sende den samme typen til backend, siden innholdet er likt(man endrer sluttdato). Men grunnen til at det fremstilles annerledes er at ordlyden av "Forleng deltakelse" blir feil dersom deltakeren allerede har sluttet på tiltaket. Vinduet som dukker opp i frontend ser også annerledes ut på denne typen.

jeg er ikke sikker på om vi burde bruke samme endringsmelding i backend på de to, men, endte på å gjøre de forskjellig fordi endringsmeldinger er arkivverdige og derfor tenkte jeg at om det er Forleng eller Endre slutttdato kanskje har en litt forskjellig funksjonell betydning som man vil dokumentere at tiltaksarrangør har gjort